packages/generated/deployments/alpha/river/addresses/riverRegistry.json

```diff
@@ -0,0 +1 @@
+{"address":"0xD85bB9e2CEcdA2b8Bf39aBcb514C222046029951"}
```

Comment:
@giuseppecrj why does this contract address show up as "multi-init" deployed 6 months ago? 

https://base-sepolia.blockscout.com/address/0xD85bB9e2CEcdA2b8Bf39aBcb514C222046029951